common/app/views/support/HtmlCleaner.scala

```diff
@@ -941,9 +934,7 @@ object AffiliateLinksCleaner {
     if (!contentHasAlwaysOffTag(tagPaths, alwaysOffTags) && switchedOn) {
       if (showAffiliateLinks.isDefined) {
         showAffiliateLinks.contains(true)
-      } else {
-        supportedSections.contains(section) && !tagPaths.exists(path => defaultOffTags.contains(path))
-      }
+      } else false
     } else false
```

Comment:
This makes it much clearer to understand, just confirming my understanding that the logic now defaults to not showing affiliate links if showAffiliateLinks is not defined?

Reply:
Yep that's correct. In the cases that we want to show affiliate links, the value showAffiliateLinks should be defined. Defaulting to false is safer and shouldn't cause issues where we do want to see affiliate links